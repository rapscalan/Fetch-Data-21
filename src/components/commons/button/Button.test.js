import React from 'react';
import { shallow } from 'enzyme';
import Button from './Button';

describe('Button component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Button text="My Button" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('matches a snapshot with type prop passed', () => {
    const wrapper = shallow(<Button type="button" text="My Button" />);
    expect(wrapper).toMatchSnapshot();
  });
});

