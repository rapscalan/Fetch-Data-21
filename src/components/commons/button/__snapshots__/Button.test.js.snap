// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component matches a snapshot 1`] = `
<button>
  My Button
</button>
`;

exports[`Button component matches a snapshot with type prop passed 1`] = `
<button
  type="button"
>
  My Button
</button>
`;
